catalyst_voices/packages/internal/catalyst_voices_services/lib/src/registration/registration_service.dart

```diff
@@ -172,8 +172,11 @@ final class RegistrationServiceImpl implements RegistrationService {
     }
 
     // TODO(damian-molinski): should come from backend
-    const displayName = 'Recovered';
     const email = 'recovered@iohk.com';
+    final catalystIdUri = Uri.parse(
+      'id.catalyst://recovered@preprod.cardano/FftxFnOrj2qmTuB2oZG2v0YEWJfKvQ9Gg8AgNAhDsKE=',
```

Comment:
The base64 must be without padding:

```suggestion
      'id.catalyst://recovered@preprod.cardano/FftxFnOrj2qmTuB2oZG2v0YEWJfKvQ9Gg8AgNAhDsKE',
```